### PR TITLE
refactor(bajour): my feedback for the bajour archive

### PR DESCRIPTION
**Pull request by Itrulia**
1. Changed the slider to use custom rendering -> no more !important
2. Changed to use grid-areas for no more empty divs
3. Some slight styling changes

### CODE REVIEW

File: apps/bajour/src/components/bajour/archive/archive-slider.tsx [hunk 1]
```diff
@@ -1,71 +1,36 @@
...
+import {DOMAttributes, useState} from 'react'
 
 type ArchiveSliderProps = {
-  teasers: ArchiveProps['teasers']
-  setTeaser: (teaser: ApiV1.ArticleTeaser | ApiV1.PeerArticleTeaser) => void
+  teasers: BuilderTeaserGridBlockProps['teasers']
+  setTeaser: (teaser: ApiV1.Teaser) => void
 }
 
-export const SliderContainerBig = styled('div')`
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive-slider.tsx:15]: wasn't used

File: apps/bajour/src/components/bajour/archive/archive-slider.tsx [hunk 2]
```diff
@@ -1,71 +1,36 @@
...
-`
-
-const SliderInnerContainer = styled('div')``
-
 const SliderContainer = styled('div')`
   width: 100%;
   max-width: 100vw;
   overflow: hidden;
-
-  .keen-slider {
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive-slider.tsx:43]: not needed anymore with custom rendering

File: apps/bajour/src/components/bajour/archive/archive-slider.tsx [hunk 3]
```diff
@@ -1,71 +1,36 @@
...
-  ${({theme}) => theme.breakpoints.up('md')} {
-    padding-top: ${({theme}) => theme.spacing(5)};
-  }
-
-  ${({theme}) => theme.breakpoints.up('xl')} {
-    padding-top: ${({theme}) => theme.spacing(7)};
-  }
-`
-
-const SliderInnerContainer = styled('div')``
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive-slider.tsx:36]: wasn't needed

File: apps/bajour/src/components/bajour/archive/archive-slider.tsx [hunk 4]
```diff
@@ -131,57 +96,59 @@ const imageStyles = css`
   object-fit: cover;
 `
 
-const getDate = (date: string) => {
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive-slider.tsx:134]: already present in website builder

File: apps/bajour/src/components/bajour/archive/archive.tsx [hunk 5]
```diff
@@ -1,32 +1,27 @@
...
   selectTeaserLead,
   selectTeaserTitle,
   selectTeaserUrl,
   useWebsiteBuilder
 } from '@wepublish/website'
-import Image from 'next/image'
 import {useState} from 'react'
 
-import Logo from '../../../logo.svg'
+import {ReactComponent as Logo} from '../../../logo.svg'
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive.tsx:12]: using SVGO

File: apps/bajour/src/components/bajour/archive/archive.tsx [hunk 6]
```diff
@@ -1,32 +1,27 @@
...
 import {ArchiveSlider} from './archive-slider'
 
-export type ArchiveProps = Omit<BuilderTeaserGridBlockProps, 'teasers'> & {
-  teasers: ApiV1.ArticleTeaser[] | ApiV1.PeerArticleTeaser[]
```
Comment (Itrulia) [inline apps/bajour/src/components/bajour/archive/archive.tsx:18]: We don't know the type of teaser. Using teaser functions means we can also show pages or events if wanted

File: apps/bajour/src/components/bajour/archive/archive.tsx [hunk 7]
```diff
@@ -60,60 +55,51 @@ const BajourLogo = styled(Image)`
...
 
 const Timeline = styled('div')`
   position: relative;
-  font-weight: 600;
-  display: grid;
-  color: ${({theme}) => theme.palette.error.main};
   grid-column: 3/7;
   grid-row: 2/3;
+  display: grid;
+  grid-template-areas:
```
[the remark this answers is not in the document]
Comment (antkiewiczk) [reply apps/bajour/src/components/bajour/archive/archive.tsx:68]: 👌

**Review by antkiewiczk — APPROVED** [review]

**Pull request merged**